src/Razor/src/Microsoft.VisualStudio.RazorExtension/RazorPackage.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.VisualStudio.RazorExtension
     [ProvideEditorFactory(typeof(RazorEditorFactory), 101, CommonPhysicalViewAttributes = (int)__VSPHYSICALVIEWATTRIBUTES.PVA_SupportsPreview)]
     [PackageRegistration(UseManagedResourcesOnly = true)]
     [AboutDialogInfo(PackageGuidString, "ASP.NET Core Razor Language Services", "#110", "#112", IconResourceID = "#400")]
+    [ProvideBindingPath]
```

Comment:
This adds the binding path to all assembly loads in VS, which has a negative perf impact and is therefore discouraged.  The preferable (if onerous) solution is to use `[ProvideCodeBase(...)]` or [`ProvideBindingRedirect(...)]` for each assembly.

Reply:
Ahhh, awesome. Thanks for the heads up Jimmy. I"ll definitely do that.